sql/migrations/20200627204553_world.sql

```diff
@@ -218,18 +218,23 @@ INSERT INTO `creature_movement_special` (`id`, `point`, `position_x`, `position_
 (58302, 8, -9295.63, 392.504, 76.2484, 100, 0, 0, 0),
 (58302, 9, -9300.58, 404.784, 74.2369, 100, 2000, 0, 0);
 
--- add stormwind guard 79861 waypoints
+
+-- add missing guard
+INSERT INTO `creature` (`guid`, `id`, `id2`, `id3`, `id4`, `map`, `display_id`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`, `wander_distance`, `health_percent`, `mana_percent`, `movement_type`, `spawn_flags`, `visibility_mod`, `patch_min`, `patch_max`) VALUES
+(99140, 1423, 0, 0, 0, 0, 0, 0, -9073.3, 427.18, 93.0558, 0.887339, 285, 285, 0, 100, 0, 2, 0, 0, 0, 10);
+
+-- add stormwind guard 99140 waypoints
 UPDATE `creature` SET `position_x` = -9073.3, `position_y` = 427.18, `position_z` = 93.0558, `movement_type` = 2 WHERE `guid` = 79861;

```

Comment:
You changed guid in waypoints below, but not in update here. Is that intended?

Reply:
Not intended, my mistake cheers for reviewing.